quic/transport/tlsbackend.nim

```diff
@@ -1,15 +1,26 @@
 import results
-import ngtcp2/native/picotls
 import ngtcp2
+import ./ngtcp2/native
 import ../errors
 
+export CertificateVerifier
+export certificateVerifierCB
+export CustomCertificateVerifier
+export InsecureCertificateVerifier
+export init
+export destroy
+
 type TLSBackend* = ref object
   picoTLS*: PicoTLSContext
 
 proc init*(
```

Comment:
would it make sense to have `newClient(...)` and `newServer()` instead of `isServer` flag?

Reply:
Done in https://github.com/vacp2p/nim-quic/pull/56/commits/c3e770959ed02a6d53cf978db4485f7dcce1a473